shared/styles/index.desktop.tsx

```diff
@@ -150,7 +150,7 @@ export const collapseStyles = (styles: ReadonlyArray<T.CollapsibleStyle>): Objec
     (a: Array<T.CollapsibleStyle>, e: T.CollapsibleStyle) => a.concat(e),
     []
   ) as Array<Object | null | false>
-  const style = flattenedStyles.reduce<Object>((o, e) => (e ? {...o, ...e} : o), {})
+  const style = flattenedStyles.reduce<Object>((o, e) => Object.assign(o, e), {})
```

Comment:
Isn't `(o, e) => Object.assign(o, e)` just `Object.assign`

Reply:
its very dangerous to pass along functions from one place to another w/o constraint. for example here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Syntax
Object.assign  takes arbitrary params and merges them, so you can do
`Object.assign(a,b,d,e,f)` etc
what does reduce pass (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#Syntax)
`accumulator, currentValue[, index[, array]])[, initialValue]`, meaning if we just passed Object.assign to reduce it'd pass alllllllll those params in, which isn't what we want.

This pattern happens elsewhere. It's better to just be explicit about it